javascript/core/intro-js/javascript-basics-part-2.md

```diff
@@ -26,68 +15,79 @@ aspects:
 ---
 ## Content
 
-Like most programming languages, JavaScript has a number of syntax rules that must be adhered to.
+Like most programming languages, JavaScript has a number of syntax rules that it must adhere to.
 
 ### Semicolon
 
-In ECMAScript it is best practice to add a semicolon at the end of a statement:
+Statements in JavaScript end in a `;`. Based on the ECMAScript[1] rules however, adding a `;` isn't actually required (although it is common practice):
 
 ```javascript
-var company="enki";
+let firstName = "Enki";
+//                    ^
 ```
 
-It’s not strictly necessary to do this as the parser will try to determine the end of a statement and add a semicolon automatically. However, it will help you be sure that the code runs as expected because automatic semicolon insertion sometimes causes errors.
+Omitting a semicolon in certain situations can cause problems. JavaScript has a feature called Automatic Semicolon Insertion (ASI) which means that, if you omit a semicolon, JavaScript will automatically add it where it thinks it should go. 
+
+This can sometimes lead to unexpected results[2], thus it's usually recommended to always insert the `;` yourself.
```

Comment:
what do you think of this?
```suggestion
This can sometimes lead to unexpected results[2]. Thus, it's usually recommended to insert the `;` yourself.
```